src/Config/container.ts

```diff
@@ -59,12 +59,17 @@ import {
   DefaultRuntimeDetector
 } from '../Repeater';
 import { container, Lifecycle } from 'tsyringe';
+import path from 'path';
```

Comment:
Don't use modules as namespaces

```suggestion
import { dirname } from 'path';
```